Lps 108285

laszlopap opened the pull request:
Hi @ericyanLr!

Thanks for the help on the previous PRs.

I am not familiar with this component, and just wanted to help the other TSEs, who had dozens of issues :)

Please review.

Thanks!

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

ericyanLr commented:
Hi @laszlopap ,

I took a closer look at noticed that there's logic similar to the one you proposed for: [ServerSummaryStatistics.java](https://github.com/ericyanLr/liferay-portal/pull/153/commits/904cddcf905cb9ce911008e2c72343a6900c9c31#diff-05bfd51dce1145f57456f76ae2727652R51-R55)

For instance, [ActionRequestSummaryStatistics.java](https://github.com/ericyanLr/liferay-portal/pull/153/commits/904cddcf905cb9ce911008e2c72343a6900c9c31#diff-b7647e52327f678be7dfb12a562449cdL77-L89) also has similar logic. A few of the other related classes also include similar logic as well.

From my understanding, these may possibly run into the same issue as well. Since they are within the same module and related classes, I think it would be reasonable to handle these as well.

Please let me know your thoughts and if you have any questions.

Best,
Eric

laszlopap commented:
Hi @ericyanLr ,

I have also updated  ActionRequestSummaryStatistics.java in this PR.

In this PR I have updated:
-ServerSummaryStatistics.java
-ActionRequestSummaryStatistics.java
-EventRequestSummaryStatistics.java
-RenderRequestSummaryStatistics.java

Do I miss other related classes?

Thanks for the help!

Regard,
László

ericyanLr commented:
Hi @laszlopap ,

From a quick glance, I would also consider taking a look at ResourceRequestSummaryStatistics.java

**Do I miss other related classes?**
I'll leave that to you, the observations I made from my previous comments were mainly from doing quick searches within the portal-monitoring module.

Hope that helps!

Best,
Eric

laszlopap commented:
Hi @ericyanLr ,

Sorry, I forgot to mention that I have updated the logic in ResourceRequestSummaryStatistics.java ,too. 

However I did not find any other java file in the portal-monitoring module which has the same logic.

Could you please review this?

Thanks,
László

ericyanLr commented:
Hi @laszlopap ,

Regarding: 
> Could you please review this?

I'm not sure what you're referring to, but, feel free to resend a pr.

Best,
Eric
